readthedocs/analytics/models.py

```diff
@@ -19,6 +21,29 @@ def _last_30_days_iter():
     return (thirty_days_ago + timezone.timedelta(days=n) for n in range(31))
 
 
+class PageViewManager(models.Manager):
+    def register_page_view(self, project, version, path, status):
+        # Normalize path to avoid duplicates.
+        path = path.strip("/")
+        if not path:
+            path = "/"
+        fields = dict(
```

Comment:
Is there a reason we're using fields here, instead of just directly in the `get_or_create`? Probably worth a comment if so, so we don't try to change it in the future.